requirements.txt

```diff
@@ -23,7 +23,7 @@ mypy_extensions==0.4.1
 # docs
 sphinx==2.1.2
 sphinx-gallery==0.4.0
-sphinx_rtd_theme=0.4.3
+sphinx_rtd_theme==0.4.3
```

Comment:
could be this why documentation doesn't work with python 3.6 ?

Reply:
It wasn't that. It should be just a typo or it could not pass the static check by pip.

The doc can be built on my local machine by adding the module path with
```python
sys.path.append(current_directory)
```.

But no luck on the 3.6 CI. Can you test on your machine if possible?

Thanks,
Jian

Reply:
Will do during the day